### PR TITLE
Gun sprite tweaks

### PR DESCRIPTION
**Pull request by Alliostra**
## About The Pull Request

- Cell overlay for the Brigador

- you can now adjust whether the onsuit or the back sprite of a gun will be displayed, because there's  many back slot icons that are rarely used because the back slot is occupied by packs 99% of the time

- Many back-exclusive sprites are now also used as the onsuit sprites and vice versa

The kammerer and clarissa are also no longer called shotgun pump and self-loading pistol.

## Why It's Good For The Game

more sprites being used = good

## Changelog
:cl:
add: cell overlay for the brigador
add: you can now adjust the way a gun will be displayed when worn on your suit and back slot (will only work for guns with differing back and suit slot sprites)
tweak: turns most back gun sprites into onsuit sprites as well and vice versa.
/:cl:

### CODE REVIEW

**Review by TheShown911 — APPROVED** [review]

File: code/datums/autolathe/guns.dm [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 /datum/design/autolathe/gun/clarissa
 	name = "FS HG .35 \"Clarissa\""
-	build_path = /obj/item/weapon/gun/projectile/selfload/clarissa
+	build_path = /obj/item/weapon/gun/projectile/selfload/
```
Comment (drexample) [inline code/datums/autolathe/guns.dm:20]: remove the / at the end

**Review by drexample — DISMISSED** [review]

**Comment by Evankhell561** [comment]
:cry:

**Review by mattroks101 — APPROVED** [review]
Code looks good.

**Comment by TheShown911** [comment]
conflicts

**Pull request merged**